_config/site.yml

```diff
@@ -139,7 +139,7 @@ profiles:
     dcp2_base_protocol_relative_url: //dcp.stage.jboss.org/
     keycloak_account_url: https://it-developers.stage.redhat.com/auth/realms/rhd/account/
     keycloak_auth_url: https://it-developers.stage.redhat.com/auth/
-    metrics: true
+    metrics: false
```

Comment:
We need metrics on for this build. Notice that it's using the staging version of the dtm_code